### DISCUSSION TITLE
__init__ and WEB_DIRECTORY

### DISCUSSION THREAD

**Discussion by chrisgoringe** [rgthree/rgthree-comfy#63] [General]
noted this in __init__.py:
```python
# This doesn't import correctly..
# WEB_DIRECTORY = "./web"
```

Just checking that you also did
```python
__all__ = ['NODE_CLASS_MAPPINGS', 'WEB_DIRECTORY']
```

because it took me a while to realize that was needed (and WEB_DIRECTORY now works for me...)

**Comment by rgthree** [comment]
Ah, awesome. Thanks, that fixed it!
